plugins/base/src/main/kotlin/translators/descriptors/DefaultDescriptorToDocumentableTranslator.kt

```diff
@@ -430,7 +430,10 @@ private class DokkaDescriptorVisitor(
         originalDescriptor: PropertyDescriptor,
         parent: DRIWithPlatformInfo
     ): DProperty {
-        val (dri, inheritedFrom) = originalDescriptor.createDRI()
+        val (dri, inheritedFrom) = originalDescriptor.createDRI().let {
+            if (it.second != null) (it.second to it.first) as Pair<DRI, DRI?>
+            else it
+        }
```

Comment:
Most tricky part. I believe this change was a little https://github.com/Kotlin/dokka/commit/37243ad7a5152d4f0f30bb8f1446d4d5c6ce2094 was not so accurate (In case of B extending A, I was getting `DRI` for A class and `inheritedFrom` for B class.)

Reply:
Actually, my implementation was also wrong, see newer commit